intellij/src/saros/intellij/editor/annotations/AnnotationManager.java

```diff
@@ -687,24 +688,24 @@ private RangeHighlighter addRangeHighlighter(
       return null;
     }
 
-    Color color;
+    // Retrieve color keys based on the color ID selected by this user. This will automatically
+    // fall back to default colors, if no colors for the given ID are available.
+    final ColorKeys colorKeys = ColorManager.getColorKeys(user.getColorID());
```

Comment:
I noticed the discrepancy of the `User#getColorID()` method name casing and the `ColorKeys#getId()` method. According to the Google Java stylebook, `getId()` would be the correct name: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Is this divergence from the style intentional, or is it an outdated format? Should I fix the name of the `ColorKeys#getId()` method, or should we keep it and create an issue for fixing the remaining code?